readme.md

```diff
@@ -175,6 +175,37 @@ Default: `process.cwd()`
 
 [Inform iTerm2](https://www.iterm2.com/documentation-escape-codes.html) of the current directory to help semantic history and enable [Cmd-clicking relative paths](https://coderwall.com/p/b7e82q/quickly-open-files-in-iterm-with-cmd-click).
 
+### iTerm.annotation(message, options?)
+
+#### message
+
+Type: `string`<br>
+
+The annotation contents.
```

Comment:
The text in readme and the TypeScript definition should be in sync. This makes it easier for me to keep both up to date.